src/interface/web/app/settings/page.tsx

```diff
@@ -1162,18 +853,13 @@ export default function SettingsView() {
                                             </Card>
                                         </div>
                                     </div>
-                                    {isManageFilesModalOpen && (
-                                        <ManageFilesModal
-                                            onClose={() => setIsManageFilesModalOpen(false)}
-                                        />
-                                    )}
                                     <div className="section grid gap-8">
                                         <div className="text-2xl">Content</div>
                                         <div className="cards flex flex-wrap gap-16">
                                             <Card id="computer" className={cardClassName}>
                                                 <CardHeader className="flex flex-row text-2xl">
```

Comment:
Should the text size be xl of the card title? Other cards seem to be `text-xl` too

```suggestion
                                                <CardHeader className="flex flex-row text-xl">
```